Formula/crystal-lang.rb

```diff
@@ -57,11 +56,11 @@ def install
     ENV["CRYSTAL_CONFIG_PATH"] = prefix/"src:lib"
     ENV.append_path "PATH", "boot/bin"
 
+    system "make", "deps"
+    (buildpath/".build").mkpath
     if build.with? "release"
-      system "make", "crystal", "release=true"
+      system "bin/crystal", "build", "--release", "--no-debug", "-o", ".build/crystal", "src/compiler/crystal.cr"
```

Comment:
Please let's just merge the release and anon-release branches to just add `--release` nor not. We should absolutely not be missing `-D without_openssl` and `-D without_zlib` on release builds.

Reply:
FYI no one is using `--without-release` according to our analytics so perhaps we should just remove the option. The same is true for `--without-shards`.